src/index.html

```diff
@@ -36,6 +28,11 @@
     You need to enable JavaScript to run this app.
   </noscript>
   <div class="xrpl-explorer" id="xrpl-explorer"></div>
+  <script type="module">
+    import { Buffer } from "buffer";
+    window.Buffer = Buffer;
```

Comment:
You shouldn't need this with the setup you have in `vite.config`. Was it causing problems without this script?